File: tests/fixtures/csharp/complex_methods.cs
public class OrderProcessor
{
    public string ProcessOrder(int status, bool verified, bool inStock,
                               bool shipped, bool cancelled, bool delivered)
    {
        if (status == 0)
        {
            if (verified)
            {
                if (inStock)
                {
                    status = 1;
                }
                else
                {
                    status = 2;
                }
            }
            else
            {
                status = 3;
            }
        }
        else if (status == 1)
        {
            if (shipped)
            {
                status = 4;
            }
            else if (cancelled)
            {
                status = 5;
            }
        }
        else if (status == 2)
        {
            if (verified && inStock)
            {
                status = 1;
            }
            else if (cancelled)
            {
                status = 5;
            }
        }
        else if (status == 4)
        {
            if (delivered)
            {
                status = 6;
            }
        }
        return status.ToString();
    }
}

File: tests/fixtures/swift/complex_methods.swift
func processOrder(status: Int, verified: Bool, inStock: Bool, shipped: Bool, cancelled: Bool, delivered: Bool, returned: Bool, refunded: Bool) -> String {
    if status == 0 {
        if verified {
            if inStock {
                return "ready"
            } else {
                return "out_of_stock"
            }
        } else {
            return "unverified"
        }
    } else if status == 1 {
        if shipped {
            return "shipped"
        } else if cancelled {
            return "cancelled"
        } else {
            return "pending"
        }
    } else if status == 2 {
        if delivered {
            return "delivered"
        } else if returned {
            return "returned"
        }
    } else if status == 3 {
        if refunded {
            return "refunded"
        }
    }

    switch status {
    case 10:
        return "archived"
    case 11:
        return "flagged"
    case 12:
        return "escalated"
    case 13:
        return "resolved"
    default:
        return "unknown"
    }
}
